community.html

```diff
@@ -42,17 +42,9 @@ <h3>IETF</h3>
 
           <p>Since presenting the Interledger project at IETF Berlin, the IETF have established another mailing list for the IETF community interested in the project; <a href="https://www.ietf.org/mailman/listinfo/ledger">ledger@ietf.org</a></p>
 
-          <h2>Gitter</h2>
+          <h2>Slack</h2>
 
-          <p>Want to chat about Interledger or have a question to ask? Join one of the rooms on Gitter:</p>
-
-          <ul>
-            <li>Project Home: <a href="https://gitter.im/interledger/Lobby">https://gitter.im/interledger/Lobby</a></li>
-            <li>Connector Operators: <a href="https://gitter.im/interledger/connector.land">https://gitter.im/interledger/connector.land</a></li>
-            <li>Ledger Plugin Developers: <a href="https://gitter.im/interledger/ledger-plugins">https://gitter.im/interledger/ledger-plugins</a></li>
-            <li>Java Development: <a href="https://gitter.im/interledger/java">https://gitter.im/interledger/java</a></li>
-            <li>Spec discussions: <a href="https://gitter.im/interledger/rfcs">https://gitter.im/interledger/rfcs</a></li>
-          </ul>
+          <p>Want to chat about Interledger or have a question to ask? <a target="_blank" href="https://communityinviter.com/apps/interledger/interledger-working-groups-slack">Join on Slack</a></p>
```

Comment:
Can you mention that most of the channels are for working groups focused on specific topics but that people can ask general questions in the #questions-and-answers channel?